src/Sportalytics.Event.Domain/Entities/ApiSports/Fixture.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Sportalytics.Event.Domain.Entities.ApiSports;
+
+public class Fixture
+{
+    [JsonPropertyName("id")]
```

Comment:
On JsonObject attribute with camelCase naming policy on class level everywhere

Reply:
I want to have uppercase for naming.